mindone/models/latte.py

```diff
@@ -81,8 +84,8 @@ def __init__(
             )
         num_patches = self.x_embedder.num_patches
         # Will use fixed sin-cos embedding:
-        self.pos_embed = ms.Parameter(ops.zeros((1, num_patches, hidden_size), dtype=ms.float32), requires_grad=False)
-        self.temp_embed = ms.Parameter(ops.zeros((1, num_frames, hidden_size), dtype=ms.float32), requires_grad=False)
+        self.pos_embed = ops.zeros((1, num_patches, hidden_size), dtype=ms.float32)
```

Comment:
just change to `self.pos_embed = Tensor(pos_embed).float().unsqueeze(0)` here for easier reading